.gitignore

```diff
@@ -34,7 +34,6 @@ sunpy/data/sample/*.fit
 
 ### https://raw.github.com/github/gitignore/master/Global/OSX.gitignore
 
-.DS_Store
```

Comment:
@dpshelio - .DS_Store is already in.
@wafels - you're right, I didn't think about checking!